README.md

```diff
@@ -168,11 +312,83 @@ Now abandon the assumption of valid arguments and program defensively!  Rewrite
 
 You may use the provided [template file](cards-template1b.js), which has extra tests checking the results of invalid arguments.
 
```

Comment:
You've got the right idea here, but your placement of the check is just a bit off. Ideally you have a helper function, which your rank function can call upon to verify the id it is given is accurate. Otherwise, you would want your if statement (line 315) to live inside your rank function. This will apply to problems below as well. Otherwise, you're checking for the correct range of values, and making sure that the id type is number, so well done!